src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -38,8 +37,12 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
             |Options:
             |""".stripMargin)
 
-
-  implicit val dateTimeConv: ValueConverter[DateTime] = singleArgConverter[DateTime](conv = DateTime.parse)
+  val httpUrl: ValueConverter[URL] = singleArgConverter[URL](s => {
+    val result = new URL(s)
+    if(result.getProtocol == null || !result.getProtocol.startsWith("http"))
```

Comment:
Why not an exact match on "http" (and "https", I guess, that should be looked up in the Fedora docs). The URL parses the URL for you. It seems a waste not to use that.